lib/shopify_cli/services/app/create/rails_service.rb

```diff
@@ -108,7 +108,7 @@ def check_dependencies
           end
 
           def check_ruby
-            ruby_version = Environment.ruby_version(context)
+            ruby_version = Environment.ruby_version(context: context)
```

Comment:
Why is this change here if it's already in main? 🤔 

https://github.com/Shopify/shopify-cli/blob/750a19af9da79b173860afe7e190371ce0e85aee/lib/shopify_cli/services/app/create/rails_service.rb#L111